modules/db/src/main/scala/gem/dao/ProgramDao.scala

```diff
@@ -114,20 +114,20 @@ object ProgramDao {
       case x => sys.error("Unexpected Option[Step] inputs: " + x)
     }, _ => sys.error("decode only"))
 
-  private implicit val OptionObservationOptionStep: Composite[Option[(Observation[Nothing], Option[Step[_]])]] =
-    Composite[(Option[Observation.Id], Option[String], Option[Step[_]])].xmap({
-      case (None,      None,        None) => None
-      case (Some(oid), Some(title), step) => Some((Observation(oid, title, None, Nil), step))
+  private implicit val OptionObservationOptionStep: Composite[Option[(Observation[Nothing], Option[Sequence.Id], Option[Step[_]])]] =
```

Comment:
Rob would have a more detailed answer but this enables Doobie to figure out how to map query output where we are expecting that type: `Option[(Observation[Nothing], Option[Sequence.Id], Option[Step[_]])]`.